HighwayPatrol.h

```diff
@@ -1,11 +1,21 @@
 #pragma once 
-
 #include "Vehicle.h"
 
-struct HighwayPatrol : Vehicle 
+struct HighwayPatrol : public Vehicle 
 {
     HighwayPatrol();
+
+    //virtual ~HighwayPatrol() = default;
+    HighwayPatrol(const HighwayPatrol&) = default;
+    HighwayPatrol& operator=(const HighwayPatrol&) = default;
+
     void scanHighway(Highway* h);
 
     void pullOver( Vehicle* v, bool willArrest, Highway* h );
-};
\ No newline at end of file
+
+    int checkSpeed(Vehicle* v);
+
+    //needed a virtual function to get rid og the warning.
+    void setSpeed(int s) override;
```

Comment:
You forgot to implement the destructor.
CheckSpeed does not need to exist.
From the instructions:
    9) clear any warnings as best you can, based on what you've learned in the previous projects
        see the note in main() about implementing the special member functions.

The destructor is a special member function.
- [ ] revise